src/siteSettings/views/index.tsx

```diff
@@ -139,6 +139,12 @@ export const SiteSettings: React.FC<SiteSettingsProps> = ({ params }) => {
                           }
                         }
                       });
+
+                      return [
```

Comment:
Just to make sure, these errors shouldn't be spread like in `MenuDetails/index.tsx` ?

Reply:
They should!